src/components/Branding/styled-components.js

```diff
@@ -0,0 +1,21 @@
+import styled from 'styled-components';
+import { color } from '../common-styled-components';
+
+export const BrandingStyles = styled.div`
+  overflow-wrap: break-word;
+  top: 8px;
+  position: fixed;
+  text-align: center;
+  font-size: 28px;
+  letter-spacing: -2px;
+  span:nth-child(1) {
+    color: ${color.blue};
+    font-weight: bold;
+  }
+  span:nth-child(2) {
+    color: ${color.orange};
+    font-weight: bold;
+  }
+`;
+
+export const BrandingContainer = styled.div``;
```

Comment:
If your outer container is just an empty div, why not just move BrandingStyles into BrandingContainer?